spec/index.html

```diff
@@ -1821,8 +1821,16 @@ <h3>Mapping Update Requests to the Formal Model</h3>
     </section>
     <section id="conformance">
       <h2>Conformance</h2>
-      <p>See <a href="#grammar">appendix B SPARQL 1.2 Update Grammar</a> regarding conformance of SPARQL Update strings.</p>
-      <p>This specification is intended for use in conjunction with: the [[[SPARQL12-GRAPH-STORE-PROTOCOL]]] and the [[[SPARQL12-PROTOCOL]]].</p>
+      <p>See 
+        <a data-cite="SPARQL12-QUERY#dfn-sparql-update-string">SPARQL update string</a>
+        and the <a data-cite="SPARQL12-QUERY#grammar">SPARQL grammar</a>
+        regarding the SPARQL update language.
```

Comment:
Capitalization taken from the linked document. "Update Grammar" is only capitalized there when it's in a title; "Update String" and "Update Language" are capitalized within prose blocks.
```suggestion
      <p>See 
        <a data-cite="SPARQL12-QUERY#dfn-sparql-update-string">SPARQL Update String</a>
        and the <a data-cite="SPARQL12-QUERY#grammar">SPARQL grammar</a>
        regarding the SPARQL Update Language.
```

Reply:
The PR on the other document has "SPARQL update string".

Reply:
I think `SPARQL Update string` is a more sensible parallel to `RDF string`, because [`This document describes SPARQL 1.2 Update, an update language for RDF graphs`](https://w3c.github.io/sparql-update/spec/#abstract).

(I do find `SPARQL 1.2 Update`, etc., to be clumsier than `SPARQL Update 1.2`, etc., and it may be worth revisiting these titles.)

Reply:
Upper-then-lower looks odd. It's one phrase.

For the change of document title, you'll need to take that to the chairs. It's in the charter.

Reply:
> Upper-lower in the terminology looks odd. It's one phrase.

`SPARQL update` (or `SPARQL 1.2 update`) doesn't exist. 
`SPARQL Update` or `SPARQL 1.2 Update` does.

Reply:
See w3c/sparql-query#190